adapters/appnexus/appnexustest/supplemental/multi-bid.json

```diff
@@ -134,6 +134,7 @@
             "crid": "29681110",
             "w": 300,
             "h": 250,
+            "cat": ["IAB20-3"],
```

Comment:
we should add video specific cases, no brand returned , failure of mapping file load etc.

Reply:
sounds good, I will look into more detail on how the tests work, hopefully I can set up mockResponse and not have to create any new placements in console to simulate getting a video response back for the unit tests.

Reply:
ok I've added a couple of new JSON files in appnexustest/exemplary folder that use video ad type, one has a valid brand_id value and the other has an invalid one (for testing case where entry is not found in category map). When it's found expected result is bid.Cat[0] contains the IAB category, when it's not found that field should not get set, which is currently how it works so we should be good there.